lib/ret_web/controllers/api/v1/hub_controller.ex

```diff
@@ -1,16 +1,101 @@
 defmodule RetWeb.Api.V1.HubController do
   use RetWeb, :controller
+  import RetWeb.ApiHelpers
 
   alias Ret.{Hub, Scene, SceneListing, Repo}
 
   import Canada, only: [can?: 2]
+  import Ecto.Query, only: [where: 3, preload: 2]
 
   # Limit to 1 TPS
   plug(RetWeb.Plugs.RateLimit)
 
   # Only allow access to remove hubs with secret header
   plug(RetWeb.Plugs.HeaderAuthorization when action in [:delete])
 
+  @show_request_schema %{
+                         "type" => "object",
+                         "properties" => %{
+                           "created_by_account_email" => %{
```

Comment:
This should only be available if the user is an admin.

Reply:
Removed for now after conversation with Liv -- we can start with an API that gives people access to the rooms that are already visible to them under normal conditions (public, created, favorited) and add more as we get our heads around authentication (via scopes / roles / etc)